### ISSUE TITLE
Glitches in "Add a friend by email"

### ISSUE THREAD

**Issue by SailingSteve** [wevote/WebApp#1655]
One set of We Vote features works mostly, but has some glitches. Familiarize yourself with our "Add a friend by email" system and bug test it, and fix the bugs you find.  It is hard to test, since with each test we need a new email address, and the way the system deals with new accounts (as opposed to existing voters in our database) is different. So it requires careful tracking of who is sending the invite, and who is receiving the invite.

In short, on this page:
https://wevote.us/more/network

...if you click the orange "Invite Friends" button, this process works and send the correct emails.

But when a brand new the respondent accepts the friend request, they are not included as a friend until they totally refresh We Vote.

**Comment by SailingSteve** [comment]
@DaleMcGrew Is this the problem?

Voter A has an account, and invites Voter B who has never been to the site before.
Voter B gets the email invite and clicks "confirm request", which launches wevote in his browser, but on the Network page, Voter A is not listed on Voter B's "Your Friends" list, until the next time Voter B logs in.

**Comment by DaleMcGrew** [comment]
Yes, this is a bug. Is Voter B shown as signed in at that point? (They should be.) I wonder if we need to make a voterRetrieve call to update the "signed in" state? Or maybe a refresh of the friends list?

**Comment by DaleMcGrew** [comment]
We have turned off the friends section for now, and we will reopen this issue when we turn our attention to the friends section.
